airgun/entities/contentview_new.py

```diff
@@ -182,3 +193,18 @@ def step(self, *args, **kwargs):
         entity_name = kwargs.get('entity_name')
         self.parent.search(entity_name)
         self.parent.table.row(name=entity_name)['Name'].widget.click()
+
+
+@navigator.register(NewContentViewEntity, 'Version')
+class ShowContentViewVersionDetails(NavigateStep):
+    """Navigate to Content View Version screen for a specific Version."""
+
+    VIEW = ContentViewVersionDetailsView
+
+    def prerequisite(self, *args, **kwargs):
+        return self.navigate_to(self.obj, 'Edit', **kwargs)
+
+    def step(self, *args, **kwargs):
+        version = kwargs.get('version')
+        self.parent.versions.search(version)
+        self.parent.versions.table.row(version=version)['Version'].widget.click()
```

Comment:
Looks like the navigation fails here to read the versions table. I wonder if it's due to ambiguity of locators between these checkboxes 
![image](https://github.com/SatelliteQE/airgun/assets/46570670/9302a005-e2a1-413e-b05d-b73647553bfb)
or something else.

Reply:
It's actually failing to find the Version tab when trying to navigate to the Versions table period, which is even more bizarre. I did find that the locator I was using for the tab was quite ambiguous though, where it was matching at least 3 things on the page, including a breadcrumb higher up. Changing that to be very specific, seeing if it helps.

Reply:
looks like it helped, PRT passed at https://github.com/SatelliteQE/robottelo/pull/14949